Flow.Launcher/Flow.Launcher.csproj

```diff
@@ -12,6 +12,7 @@
     <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <SatelliteResourceLanguages>en</SatelliteResourceLanguages>
+    <ValidateExecutableReferencesMatchSelfContained>false</ValidateExecutableReferencesMatchSelfContained>
```

Comment:
dont think we are referencing other executables, is this needed?

Reply:
I don't know Why, but my visual studio is preventing me to build without this (the latest preview). It is possible that it is Ms issue.